cves/CVE-2013-0844.yml

```
@@ -90,16 +94,10 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: f18c873ab5ee3c78d00fdcc2582b39c133faecb4
-  note: ''
-- commit: ''
-  note: ''
+  note: Fix for vulenrability
```

```suggestion
  note: Fix for vulnerability.
```
Can you elaborate a bit more on the fix?

---

I will make sure clarify this in the yaml, but simply it involved changing a bound from "n" to "n-1"